libyul/Utilities.cpp

```diff
@@ -49,7 +49,7 @@ string solidity::yul::reindent(string const& _code)
 		auto const e = i == _s.npos ? end(_s) : next(begin(_s), static_cast<ptrdiff_t>(i));
 		auto const opening = count_if(begin(_s), e, [](auto ch) { return ch == '{' || ch == '('; });
 		auto const closing = count_if(begin(_s), e, [](auto ch) { return ch == '}' || ch == ')'; });
-		return opening - closing;
+		return int(opening - closing);
```

Comment:
This one is a bit weird... I'd actually make this ``static_cast<int>(opening) - static_cast<int>(closing)``...
It's probably a non-issue in practice, because this will only run on yul code that parses I guess - but on the other hand the case that this returns something ``< 0`` is explicitly handled below...
Then again everything being two's complements will make this behave just fine, but I'd feel better doing it differently :-)...